app/ts/types/bouquetTypes.ts

```diff
@@ -0,0 +1,33 @@
+import * as t from 'funtypes'
+import { EthereumAddress, EthereumInput, EthereumQuantity } from './ethereumTypes.js'
+
+export type TransactionList = t.Static<typeof TransactionList>
+export const TransactionList = t.ReadonlyArray(
+	t
```

Comment:
this is quite weird intendation. Why not include it in the same line with the object?

Reply:
My TS config may be borked